packages/erxes-ui/src/components/richTextEditor/extensions/Link.ts

```diff
@@ -1,10 +1,57 @@
-import TipTapLink from '@tiptap/extension-link';
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Consider using consistent naming for imports to improve readability.